src/Microsoft.AspNetCore.SignalR.Core/DefaultHubCallerContext.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading;
+using Microsoft.AspNetCore.Http.Features;
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public class DefaultHubCallerContext : HubCallerContext
+    {
+        private readonly HubConnectionContext _connection;
+
+        public DefaultHubCallerContext(HubConnectionContext connection)
+        {
+            _connection = connection;
+        }
+
+        public override string ConnectionId => _connection.ConnectionId;
+
+        public override string UserIdentifier => _connection.UserIdentifier;
+
+        public override ClaimsPrincipal User => _connection.User;
+
+        public override IDictionary<object, object> Items => _connection.Items;
+
+        public override IFeatureCollection Features => _connection.Features;
+
+        public override CancellationToken ConnectionAbortedToken => _connection.ConnectionAbortedToken;
```

Comment:
It's `RequestAborted` not `RequestAbortedToken`, so this should be `ConnectionAborted`

https://github.com/aspnet/HttpAbstractions/blob/b059bcc426ea7593ceeed2df1b6ce14fd3a626ba/src/Microsoft.AspNetCore.Http/DefaultHttpContext.cs#L158-L162

Reply:
I realize you didn't add this property, but the comment is still relevant and a simple fix ;)

Reply:
Will do